lib/survey-gizmo-ruby.rb

```diff
@@ -11,6 +11,7 @@
 require 'faraday'
 require 'faraday_middleware'
 require 'logger'
+require 'net/http'
```

Comment:
Because of: https://github.com/jarthod/survey-gizmo-ruby/blob/master/lib/survey_gizmo/connection.rb#L60

Without, an error raise saying: the `Net` namespace is missing in `SurveyGizmo::Connection` class.